src/main.rs

```diff
@@ -55,6 +56,13 @@ enum DependencySorting {
 fn main() -> Result<(), String> {
     let args = Args::parse();
 
+    println!("Linting {:?}", &args);
+
+    // Print the current working directory
+    let _cwd = std::env::current_dir()
+        .map_err(|err| format!("Could not get current directory: {:?}", err))?;
+    println!("Current directory: {:?}", _cwd);
+
```

Comment:
Not sure why we need to print the current directory?

Reply:
That is my oversight. I was using these lines to figure out why the problem was happening and forgot to remove it. I've fixed that and pushed it.